packages/components/src/lib/queries/lexical-query.js

```diff
@@ -218,18 +218,16 @@ export default class LexicalQuery extends Query {
     if (lexiconShortOpts.allow && lexiconShortOpts.allow.length > 0) {
       this.homonym.lexemes.forEach((l) => { l.meaning.clearShortDefs() })
       // Suppress events that will trigger UI messages if source is wordlist
-      if (this._source !== LexicalQuery.sources.WORDLIST) {
-        LexicalQuery.evt.HOMONYM_READY.pub(this.homonym)
-      }
+      LexicalQuery.evt.HOMONYM_READY.pub(this.homonym)
```

Comment:
maybe remove the comment above about suppressing the events since it's not the case anymore

Reply:
oh, I have missed it - fixed